doc/nrf/gs_testing.rst

```diff
@@ -21,7 +21,13 @@ How to connect with the VS Code extension
 
 The |VSC| extension is a complete IDE for developing applications for nRF91, nRF53 and nRF52 Series devices.
```

Comment:
I believe this should be:
`The |VSC| extension pack ... `

Is this right @Avalei, or is it fine as it?

Reply:
In this particular case let's add "extension pack" onto the end because the next sentence mentions nRF Terminal, so this could be a bit confusing to users.

Reply:
Added `pack` to the next sentence after this one, because IDE is always referred to as "VSC extension", not "VSC extension pack". So `pack` makes sense in the next sentence next to nRF Terminal.